BringgSDK.js

```diff
@@ -21,7 +21,16 @@ var BringgSDK = (function () {
 
   var REGIONS = {
     "ue1": "https://realtime2-api.bringg.com/",
-    "ew1": "https://eu1-realtime.bringg.com"
+    "ue2": "https://eu2-realtime.bringg.com",
+    "ew1": "https://eu1-realtime.bringg.com",
+    "us2": "https://us2-realtime.bringg.com"
+  };
+
+  var REGIONS_ENUM = {
+    'ew1': 1,
+    'ue1': 2,
```

Comment:
don't you want to change the names to eu instead of ue?

Reply:
talked about it, some tokens are using this as regions.